index.js

```diff
@@ -18,10 +36,11 @@ module.exports = {
       liveReloadOptions.snipver = 1;
     }
 
+    var liveReloadPath = buildLiveReloadPath(options.liveReloadPrefix);
```

Comment:
Hmm, can we use clean-base-url here instead? Seems like the implementation is basically the same (and that package is used a ton inside ember-cli)...

https://github.com/ember-cli/clean-base-url/blob/master/index.js